src/core/shared_library.cc

```diff
@@ -85,7 +85,14 @@ OpenLibraryHandle(const std::string& path, void** handle)
         Status::Code::NOT_FOUND, "unable to load backend library: " + errstr);
   }
 #else
-  *handle = dlopen(path.c_str(), RTLD_NOW | RTLD_LOCAL);
+  // Need to open the Python backend shared library using RTLD_GLOBAL flag so
+  // that subsequent dlopens in the user's Python model work properly
+  if (path.find("libtriton_python") != std::string::npos) {
```

Comment:
Since the stub is going to be a completely separate process now, I think this is not needed?

Reply:
Yes, this is no longer required when the stub becomes completely separate. I'll mark this PR as WIP for now.